src/app/pages/questions/containers/questions-page.component.ts

```diff
@@ -188,7 +188,7 @@ export class QuestionsPageComponent implements OnInit {
 
   exitQuestionnaire() {
     this.sendEvent(UsageEventType.QUESTIONNAIRE_CANCELLED)
-    this.navCtrl.pop()
+    this.navCtrl.pop({ animation: 'wp-transition' })
```

Comment:
Is this a wordpress (wp-) reference?

Reply:
No I think this is a Windows phone reference. I modified it because this animation style was less laggy on slower phones.